src/metabase/integrations/slack.clj

```diff
@@ -85,6 +88,16 @@
   :setter (fn [channel-name]
             (setting/set-value-of-type! :string :slack-files-channel (process-files-channel-name channel-name))))
 
+(defsetting slack-bug-report-channel
```

Comment:
we should add a backend setting for `slack-bug-reports-enabled` and have it settable only by environment variable, and use that setting to toggle visibility of the bug report channel input on the frontend